### ISSUE TITLE
Imported 13/100 devices only. devices being skipped

### ISSUE THREAD

**Issue by tanimientras** [drygdryg/netbox-device-autodiscovery#13]
Is this repo still mantained?

**Comment by tanimientras** [comment]
just added some code to detect linux systems, however storage devices or detected nas are still unrecognized...should i made a PR?
```
# Linux
if (os_class['vendor'] == 'Linux') and (os_class['osfamily'] in ('Linux')):
    return Device('Linux', 'PC', 'PC', 'Linux')
```

**Comment by spastro** [comment]
yeah i have heaps raspberry pi 3+ that are not being detected and skipped.

Is there any way to not have the script not delete devices that are found but don't match a type and maybe just tag them as unknown devices etc?
